GliaWidgets/Component/Bar/CallButtonBar/Button/CallButtonStyle.swift

```diff
@@ -1,15 +1,31 @@
 import UIKit
 
+/// Style of a button shown in call button bar.
 public struct CallButtonStyle {
+    /// Style of a button's state.
     public struct StateStyle {
+        /// Background color of the button.
         public let backgroundColor: UIColor
+
+        /// Image of the button.
         public let image: UIImage
+
+        /// Color of the image.
         public let imageColor: UIColor
+
+        /// Title of the button.
         public let title: String
+
+        /// Font of the title.
         public let titleFont: UIFont
+
+        /// Color of the title.
         public let titleColor: UIColor
     }
 
+    /// Style of active state.
```

Comment:
I probably confused it with the "icons", in the sense that if someones' audio is on or off, then the icon might be colorful or overshadowed, respectively. In case of buttons, indeed the "can be pressed" seems kind of obvious :)
On the other hand, I might not be the only one reading it with the tired head, thus let the clarification be for now.